packages/flutter_tools/test/general.shard/runner/target_devices_test.dart

```diff
@@ -1262,6 +1262,45 @@ target-device (mobile) • xxx • ios • iOS 16 (unsupported)
           expect(deviceManager.iosDiscoverer.xcdevice.waitedForDeviceToConnect, isFalse);
         });
 
+        testUsingContext('when only matching device is dev mode disabled', () async {
+          deviceManager.iosDiscoverer.deviceList = <Device>[FakeIOSDevice(deviceName: 'target-device', devModeEnabled: false)];
+
+          final List<Device>? devices = await targetDevices.findAllTargetDevices();
+
+          expect(logger.statusText, equals('''
+To use \'target-device\' for development, enable Developer Mode in Settings → Privacy & Security.
```

Comment:
info • Unnecessary escape in string literal • packages/flutter_tools/test/general.shard/runner/target_devices_test.dart:1271:8 • unnecessary_string_escapes